docs/interfaces/hdf5/index.md

```diff
@@ -62,15 +62,3 @@ with [example implementations](examples.md)
 :fontawesome-brands-github:
 [Install guide](https://github.com/KxSystems/hdf5#installation)
 
-
```

Comment:
These should be kept for each of the interfaces (although in this case updated removing reference to the beta version since it is no longer beta) i.e.

`This interface is currently available under an Apache 2.0 licence ...`